tests/e2e/operator.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir="$(dirname "$(readlink -f "$0")")"
+project_dir="$(readlink -f ${script_dir}/../..)"
+
+source "${script_dir}/lib.sh"
+
+# The operator namespace.
+readonly op_ns="confidential-containers-system"
+# There should be a registry running locally on port 5000.
+export IMG=localhost:5000/cc-operator
+
+# Build the operator and push images to a local registry.
+#
+build_operator () {
+	start_local_registry
+
+	pushd "$project_dir" >/dev/null
+	make docker-build
+	make docker-push
+	popd >/dev/null
+}
+
+# Install the operator.
+#
+install_operator() {
+	start_local_registry
+
+	# The node should be 'worker' labeled
+	local label="node-role.kubernetes.io/worker"
+	if ! kubectl get node "$(hostname)" -o jsonpath='{.metadata.labels}' \
+		| grep -q "$label"; then
+		kubectl label node "$(hostname)" "$label="
+	fi
+
+	pushd "$project_dir" >/dev/null
+	make install
+	# TODO: need any checking after install?
+	make deploy
```

Comment:
make install deploy is what developers would use and I think it is good for now. But eventually end users will install the operator in a slightly different way, i.e. from operatorhub.io using OLM (operator lifecycle manager). This is a command I  use to test this and which is close to what the end user will do. 

make IMAGE_TAG_BASE=quay.io/jensfr/confidentials-containers-operator IMG=quay.io/jensfr/confidential-containers-operator:test docker-build docker-push bundle bundle-build bundle-push catalog-build catalog-push && kubectl create -f deploy/deploy_olm.yaml

Note that deploy_olm.yaml does not exist in the git repo yet. I'll send it as a PR soon. This comment is just a heads up and shouldn't block the PR from being merged.